apps/judicial-system/web/src/routes/Court/InvestigationRequest/HearingArrangements/HearingArrangementsForm.tsx

```diff
@@ -184,10 +181,11 @@ const HearingArrangementsForm: React.FC<Props> = (props) => {
             options={registrars}
             onChange={(selectedOption: ValueType<ReactSelectOption>) =>
               setRegistrar(
-                (selectedOption as ReactSelectOption).value.toString(),
+                (selectedOption ?? undefined) &&
```

Comment:
I think this is a bit unclear. I think it would be clearer if this was a traditional if else statement i.e.

```
if (selectedOption) {
   setRegistrar(selectedOption)
} else {
   setRegistrar(undefined)
}
```